src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java

```diff
@@ -20,14 +21,16 @@ public SendEmailNotificationUseCase(
     }
 
     public void execute(UUID userId, UUID templateId) {
```

Comment:
i would return a result of the operation (success or errors)

Reply:
changed 🆙